installation_and_upgrade/define_latest_genie_python.bat

```diff
@@ -5,7 +5,7 @@ REM   LATEST_PYTHON is set to a version on genie_python that can be run
 
 set "KITS_ICP_PATH=\\isis.cclrc.ac.uk\inst$\Kits$\CompGroup\ICP"
 
-set "GENIE_DIR=%KITS_ICP_PATH%\genie_python_3"
+set "GENIE_DIR=%KITS_ICP_PATH%\genie_python_3_backup"
 
```

Comment:
do we need to use `genie_python_3_backup` ?

Reply:
We do not, this is temporary whilst I test that the changes made here https://github.com/ISISComputingGroup/genie_python/pull/351 work